btcjson/chainsvrcmds.go

```diff
@@ -80,11 +80,21 @@ func NewCreateRawTransactionCmd(inputs []TransactionInput, amounts map[string]fl
 	}
 }
 
+// ChangeType defines the different output types to use for the change address
+// of a transaction built by the node.
+type ChangeType string
+
+var (
+	ChangeTypeLegacy     ChangeType = "legacy"
+	ChangeTypeP2SHSegWit ChangeType = "p2sh-segwit"
+	ChangeTypeBech32     ChangeType = "bech32"
+)
```

Comment:
Please add comments for each of these. For example:

* ChangeTypeLegacy indicates a P2PKH change address type.
* ChangeTypeP2SHSegWit indicates a P2WPKH-in-P2SH change address type.
* ChangeTypeBech32 indicates a P2WPKH change address type.